hbp_knowledge/receptors/djouma2006.bel

```diff
@@ -74,67 +74,117 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
+UNSET Confidence
 
 SET Evidence ="antalarmin increased dopamine D2 ⁄ 3 binding in the CPu (+7%, P < 0.001; Fig. 3C) and olfactory tubercle (OT; +15%, P 1⁄4 0.006; Fig. 3D) of group-housed rats."
+SET Confidence = "Low"
 SET MeSHAnatomy = {"Neostriatum","Olfactory Tubercle"}
 a(MESH:antalarmin) -> p(RGD:Drd2)
 a(MESH:antalarmin) -> p(RGD:Drd3)
+#evidence says dopamine D2/3 binding. Should it be written as a complex of dopamine and dopamine receptors?
+UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence ="Isolation rearing caused an up-regulation of dopamine D2 ⁄ 3 receptors in both the core (+20%; F1,144 1⁄4 31.176, P < 0.001; Fig. 3A) and shell (+25%; F1,148 1⁄4 35.233, P < 0.001; Fig. 3B) of the NAcc. "
+SET Confidence = "Medium"
 SET MeSHAnatomy ="Nucleus Accumbens"
 path(HBP:"isolation rearing") -> p(RGD:Drd2)
 path(HBP:"isolation rearing") -> p(RGD:Drd3)
+UNSET MeSHAnatomy
+#MeSHAnatomy was unset
+UNSET Confidence
 
 SET Evidence ="Antalar- min treatment reversed this effect and returned dopamine D2 ⁄ 3 receptor density levels back to those of the vehicle-treated GH rats."
+SET Confidence = "High"
 a(MESH:antalarmin) -| p(RGD:Drd2)
 a(MESH:antalarmin) -| p(RGD:Drd3)
+UNSET Confidence
 
 SET Evidence ="Furthermore a significant interaction was found between housing and treatment in both the central nucleus of the amygdala (CeA; F1,139 1⁄4 11.560, P < 0.001; Fig. 3F) and basolateral amygdala (BLA; F1,137 1⁄4 7.616, P 1⁄4 0.007; Fig. 3E).Thus, isolation caused an up-regulation in dopamine D2 ⁄ 3 receptors in these regions, an effect that was reversed by antalarmin treatment (but only reached statistical significance in the CeA)."
-
+SET Confidence = "Medium"
+SET MeSHAnatomy = "Amygdala" #because the evidence indicates activities happening in amygdala
 path(HBP:"isolation rearing") -> p(RGD:Drd2)
 path(HBP:"isolation rearing") -> p(RGD:Drd3)
 a(MESH:antalarmin) -| path(HBP:"isolation rearing")
+UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence ="A significant decrease in BDNF mRNA expression resulting from ISO was found in the BLA ()25%; F1,102 1⁄4 77.937, P < 0.001), nucleus of the horizontal limb of the diagonal band (HB )75%; F1,67 1⁄4 259.106, P < 0.001) and dentate gyrus (DG )85%; F1,101 1⁄4 984.958,"
+SET Confidence = "Medium"
 SET MeSHAnatomy ={"Basolateral Nuclear Complex","Diagonal Band of Broca","Dentate Gyrus"}
 path(HBP:"isolation rearing") -| r(RGD:Bdnf)
+UNSET MeSHAnatomy #MeSHAnatomy was unset
+UNSET Confidence
 
 SET Evidence ="whilst isolation increased BDNF mRNA expression in the retrosplenial cortex"
-#SET MeSHAnatoomy =""
+SET Confidence = "Medium"
+SET Anatomy ="retrosplenial granular cortex"
 path(HBP:"isolation rearing") -> r(RGD:Bdnf)
+#MeSHAnatomy was set and unset
+UNSET Anatomy
+UNSET Confidence
 
 SET Evidence ="Treatment with antalarmin had no impact upon the isolation-induced alterations of BDNF expression."
+SET Confidence = "Medium"
+path(HBP:"isolation rearing") -> r(RGD:Bdnf)
```

Comment:
```suggestion
path(HBP:"isolation rearing") reg p(RGD:Bdnf)
```
I would go with this as only alterations in protein expression are metioned and no concrete direction of the regulation